### PR TITLE
feat: add `math/base/special/asindf`

### PR DESCRIPTION
**Pull request by gunjjoshi**
## Description

> What is the purpose of this pull request?

This pull request:

-   adds `math/base/special/asindf`, which would be the single-precision equivalent for [`math/base/special/asind`](https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/math/base/special/asind).

```c
float stdlib_base_asindf( const float x )
```

## Related Issues

> Does this pull request have any related issues?

None.

## Questions

> Any questions for reviewers of this pull request?

No.

## Other

> Any other information relevant to this pull request? This may include screenshots, references, and/or implementation notes.

Code Coverage Report:

Screenshot 2024-04-07 at 02 56 06

## Checklist

> Please ensure the following tasks are completed before submitting this pull request.

-   [x] Read, understood, and followed the [contributing guidelines][contributing].

* * *

@stdlib-js/reviewers

[contributing]: https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md

### CODE REVIEW

File: lib/node_modules/@stdlib/math/base/special/asindf/lib/main.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+* @example
+* var v = asindf( Math.sqrt( 3.0 ) / 2.0 );
+* // returns ~60.0
+*
+* @example
+* var v = asindf( NaN );
+* // returns NaN
+*/
+function asindf( x ) {
+	var rad;
```
Comment (gunjjoshi) [inline lib/node_modules/@stdlib/math/base/special/asindf/lib/main.js]: In [`asind`](https://github.com/stdlib-js/stdlib/blob/53c33cb04e6d897583ece98e24c4d98f5d87b471/lib/node_modules/%40stdlib/math/base/special/asind/lib/main.js#L56), we have declared and used `rad` in the same line, but I have declared it seperately at the top, as we always do.

Comment (gunjjoshi) [reply lib/node_modules/@stdlib/math/base/special/asindf/lib/main.js]: Also, we have an incorrect example too, in [`main.c`](https://github.com/stdlib-js/stdlib/blob/53c33cb04e6d897583ece98e24c4d98f5d87b471/lib/node_modules/%40stdlib/math/base/special/asind/src/main.c#L30) of `asind`. We might need to rectify that too.

Comment (kgryte) [reply lib/node_modules/@stdlib/math/base/special/asindf/lib/main.js]: @gunjjoshi In general, I'd be somewhat suspicious of many of the `<trig>d` functions, as those were all GSoC contributions and many of them had errors which slipped through during review. If you find errors, feel free to send PRs fixing them.

**Review by kgryte — APPROVED** [review]
LGTM. Thanks, @gunjjoshi!

**Comment by gunjjoshi** [comment]
@kgryte, I'll make appropriate changes in the upstream `asind` too. Thanks for pointing them!

**Comment by kgryte** [comment]
Thanks, @gunjjoshi! 😅

**Pull request merged**